tests/e2e/pages/webcam.page.ts

```diff
@@ -0,0 +1,17 @@
+class WebcamPage {
+    public get webCam() { return browser.element("#mxui_widget_ReactCustomWidgetWrapper_0"); }
+    public get takePictureButton() { return browser.element(".widget-camera-picture"); }
+    public get image() { return browser.getAttribute("img", "src"); }
+    public get imageElement() { return browser.element("#mxui_widget_ReactCustomWidgetWrapper_0 > img"); }
+    // public get alertConfirmation() { return browser.element("#mxui_widget_ReactCustomWidgetWrapper_0 > img");}
```

Comment:
If the code is not being used, remove it

Reply:
Done